src/core/lintString.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+/**
+ * @description wrapper for parse for programmatic use
+ * @param {String} string the text to parse
+ * @param {Object} [config] an object containing config to lint by
+ * @param {String} [filename] the name of the file
+ * @returns {Object} an object containing the complete report from linting
+ */
+var lintString = function( string, config, filename ) {
```

Comment:
Where's the connection between lintString and a file name? I thought that idea behind this API is to use it without file.